projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	Button,
+	PanelBody,
+	PanelRow,
+	SelectControl,
+	TextControl,
+	ExternalLink,
+	Placeholder,
+	Spinner,
+} from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { lock } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { API_STATE_NOT_REQUESTING, API_STATE_REQUESTING } from './constants';
+import { CURRENCY_OPTIONS } from '../../currencies';
+import { store as membershipProductsStore } from '../../../store/membership-products';
+
+export default function ProductManagementInspectorControl( {
+	allowCreateOneTimeInterval,
+	setSelectedProductId,
+} ) {
+	const siteSlug = useSelect( select => select( membershipProductsStore ).getSiteSlug() );
+	const { saveProduct } = useDispatch( membershipProductsStore );
+
+	const [ apiState, setApiState ] = useState( API_STATE_NOT_REQUESTING );
+	const [ title, setTitle ] = useState( __( 'Monthly Subscription', 'jetpack' ) );
+	const [ currency, setCurrency ] = useState( 'USD' );
+	const [ price, setPrice ] = useState( 5 );
+	const [ interval, setInterval ] = useState( '1 month' );
+
+	const intervalOptions = [
+		{ label: __( 'Month', 'jetpack' ), value: '1 month' },
+		{ label: __( 'Year', 'jetpack' ), value: '1 year' },
+	];
+	if ( allowCreateOneTimeInterval ) {
+		intervalOptions.push( { label: __( 'One-Time Payment', 'jetpack' ), value: 'one-time' } );
+	}
+
+	const handleSubmit = event => {
+		event.preventDefault();
+		setApiState( API_STATE_REQUESTING );
+		saveProduct( { title, currency, price, interval }, setSelectedProductId, success => {
+			setApiState( API_STATE_NOT_REQUESTING );
+			if ( success ) {
+				setPrice( 5 );
+				setTitle( '' );
+			}
+		} );
+	};
+
+	return (
+		<InspectorControls>
+			{ siteSlug && (
+				<ExternalLink
+					href={ `https://wordpress.com/earn/payments/${ siteSlug }` }
+					className={ 'product-management-control-inspector__link-to-earn' }
+				>
+					{ __( 'Manage your subscriptions.', 'jetpack' ) }
+				</ExternalLink>
+			) }
```

Comment:
Could we add this external link inside the "Add a new subscription" or create a "Management" panel.? It looks a bit odd to just have a link floating in the middle of the controls.
<img width="281" alt="Captura de pantalla 2022-03-19 a las 1 22 55" src="https://user-images.githubusercontent.com/1989914/159099224-b56b497b-038b-4e16-921c-120c396aec72.png">

Reply:
Added to our backlog.